docs/en/sql-reference/statements/create/table.md

```diff
@@ -227,9 +226,9 @@ hex(hexed): 5A90B714
 
 `ALIAS expr`
 
-Synonym. Such a column isn’t stored in the table at all.
-Its values can’t be inserted in a table, and it is not substituted when using an asterisk in a SELECT query.
-It can be used in SELECTs if the alias is expanded during query parsing.
+Calculated columns (synonym). Column of this type are not stored in the table and it is not possible to INSERT values into them.
+
+When SELECT queries explicitly reference columns of this type, the value is computed at query time from `expr`. By default, `SELECT *` excludes ALIAS columns. This behavior can be disabled with setting `asteriks_include_alias_columns`.
```

Comment:
L. 233 and l. 235 are badly confusing. What is "old data"? It sounds like something is actually persisted, even if l. 229 says nothing is persisted.